src/cpu/x64/jit_generator.hpp

```diff
@@ -1368,6 +1370,19 @@ class jit_generator : public Xbyak::CodeGenerator, public c_compatible {
         vcvttps2dq(x, op);
     }
 
+    void uni_vcvtsi2ss(const Xbyak::Xmm& x1, const Xbyak::Xmm& x2, const Xbyak::Operand& op) {
+        if (is_valid_isa(avx)) {
+            vcvtsi2ss(x1, x2, op);
+        } else {
+            assert(x1.getIdx() == x2.getIdx());
+            cvtsi2ss(x1, op);
+        }
+    }
+
+    void uni_vcvtsi2ss(const Xbyak::Ymm& x1, const Xbyak::Ymm& x2, const Xbyak::Operand& op) {
```

Comment:
This function is useless. The instruction *vcvtsi2ss* works with XMM only.

Reply:
done